tap_playvox/sync.py

```diff
@@ -159,10 +186,19 @@ def sync_endpoint(client,
                         
                         if len(date_records)==parsed_date_records:
                             parse = False
-                            
+
+        # Subtract 14 days from the current date and time for Schedule Metrics
+        startDate = datetime.utcnow() - timedelta(days=14)
+
+        # Set the time to midnight
+        startDate = startDate.replace(hour=0, minute=0, second=0, microsecond=0)
+                   
```

Comment:
what are you trying to achieve here?

Reply:
As part of the Playvox WFM platform, the agents can update their punch-in and punch-out times for the previous 14 days. In order to capture their actual work hours, the tap for this API needs to be run in a Rolling 14-day schedule.